frontendUiTableImpl/src/main/java/com/virtuslab/gitmachete/frontend/ui/impl/cell/BranchOrCommitCellRendererComponent.java

```diff
@@ -155,7 +154,6 @@ public BranchOrCommitCellRendererComponent(
 
       if (commitItem.getCommit().equals(forkPoint)) {
         append(" ${HEAVY_WIDE_HEADED_RIGHTWARDS_ARROW} fork point ? ", new SimpleTextAttributes(STYLE_PLAIN, Colors.RED));
-        append("(Resolve) ", LINK_ATTRIBUTES);
```

Comment:
Okay so we're resigning from the direct link to the override-FP action, right?

Reply:
![image](https://user-images.githubusercontent.com/3383210/88942140-236eb880-d28a-11ea-8def-258815a2802b.png)